core/server/data/meta/description.js

```diff
@@ -14,7 +14,7 @@ function getDescription(data, root) {
     } else if (_.contains(context, 'author') && data.author) {
         description = data.author.bio;
```

Comment:
Yeah, I realise that - but as you made a very similar change for tag (only the reverse) I thought it might be worthwhile to make the same change for authors whilst we're here, and then we know all the behaviour is straightened out :grin: